### PR TITLE
Core/Conditions: Added CONDITION_SCENE_IN_PROGRESS, useful for cases where you need phase things when a scene is in progress.

### PR DESCRIPTION
**Pull request by BertiRean**
**Changes proposed:**

-  
-  
-  

**Issues addressed:**

Closes #  (insert issue tracker number)

**Tests performed:**

(Does it build, tested in-game, etc.)

**Known issues and TODO list:** (add/remove lines as needed)

- [ ] 
- [ ]

### CODE REVIEW

File: src/server/game/Conditions/ConditionMgr.h [hunk 1]
```diff
@@ -111,6 +111,7 @@ enum ConditionTypes
     CONDITION_TYPE_MASK                = 52,                   // TypeMask         0              0                  true if object is type object's TypeMask matches provided TypeMask
     CONDITION_BATTLE_PET_COUNT         = 53,                   // SpecieId         count          ComparisonType     true if player has `count` of battle pet species
     CONDITION_SCENARIO_STEP            = 54,                   // ScenarioStepId   0              0                  true if player is at scenario with current step equal to ScenarioStepID
+    CONDITION_SCENE_IN_PROGRESS        = 55,                   // SceneScriptPackageId   0        0                  true if player is at scenario with current step equal to ScenarioStepID
```
Comment (Shauren) [inline src/server/game/Conditions/ConditionMgr.h:114]: nice comment copypasta

Comment (BertiRean) [reply src/server/game/Conditions/ConditionMgr.h:114]: Lol jaja

Comment (BertiRean) [reply src/server/game/Conditions/ConditionMgr.h]: fixed

File: src/server/game/Conditions/ConditionMgr.cpp [hunk 2]
```diff
@@ -148,6 +148,7 @@ ConditionMgr::ConditionTypeInfo const ConditionMgr::StaticConditionTypeData[COND
     { "Object TypeMask",           true, false, false },
     { "BattlePet Species Learned", true, true,  true  },
     { "On Scenario Step",          true, false, false },
+    { "On Scene In Progress",      true, false, false },
```
Comment (Shauren) [inline src/server/game/Conditions/ConditionMgr.cpp:151]:
```suggestion
    { "Scene In Progress",      true, false, false },
```

**Review by Shauren — APPROVED** [review]

File: src/server/game/Conditions/ConditionMgr.cpp (hunk 2 above)
Comment (Shauren) [inline src/server/game/Conditions/ConditionMgr.cpp:151]:
```suggestion
    { "Scene In Progress",      true, false, false },
```

**Comment by Seyden** [comment]
do you have a use case? i dont see the point in that, because everything you would do in a scene is related to the certain script event.. this means you could also just put the certain action into the script

**Comment by BertiRean** [comment]
@mdX7 @Shauren What is needed to close this?

**Pull request merged**